packages/node_modules/@webex/internal-plugin-ediscovery/test/unit/spec/content.js

```diff
@@ -199,7 +199,29 @@ describe('EDiscovery Content API Tests', () => {
     it('GetContentSummary submits an audit log request', () => {
```

Comment:
I'd be inclined to name the tests more in terms of audit log, like 
`PostAuditLog with valid reportId succeeds`
`PostAuditLog with no reportId is rejected`
`PostAuditLog propagates http error on failure`